InnerEye/Scripts/move_model.py

```diff
@@ -0,0 +1,127 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+import sys
+from pathlib import Path
+from typing import Tuple
+
+import json
+import param
+from attr import dataclass
+from azureml.core import Environment, Model, Workspace
+
+from InnerEye.Azure.azure_config import AzureConfig
+from InnerEye.Common import fixed_paths
+
+print(f"Starting runner at {sys.argv[0]}")
+innereye_root = Path(__file__).absolute().parent.parent.parent
+if (innereye_root / "InnerEye").is_dir():
+    innereye_root_str = str(innereye_root)
+    if innereye_root_str not in sys.path:
+        print(f"Adding InnerEye folder to sys.path: {innereye_root_str}")
+        sys.path.insert(0, innereye_root_str)
+
+from InnerEye.Common.generic_parsing import GenericConfig  # noqa: E402
+
+# The property in the model registry that holds the name of the Python environment
+PYTHON_ENVIRONMENT_NAME = "python_environment_name"
+MODEL_PATH = "MODEL"
+ENVIRONMENT_PATH = "ENVIRONMENT"
+MODEL_JSON = "model.json"
+
+
+@dataclass
+class MoveModelConfig(GenericConfig):
+    model_id: str = param.String(default="model_id",
+                                 doc="AzureML model_id")
+    path: str = param.String(default="path",
+                             doc="Path to import or export model")
+    action: str = param.String(default="action",
+                               doc="Import or export model from workspace. E.g. import or export")
+
+
+def get_paths(path: Path, model_id: str) -> Tuple[Path, Path]:
+    """
+    Gets paths and creates folders if necessary
+    :param path: Base path
+    :param model_id: The model ID
+    :return: model_path, environment_path
+    """
+    model_id_path = Path(path) / model_id.replace(":", "_")
+    model_id_path.mkdir(parents=True, exist_ok=True)
```

Comment:
The exist_ok=True... wouldn't it be better to say "hey, I have a model downloaded here already, that you are about to overwrite?" There could be files in that folder from an old download that would then spoil the upload

Reply:
This is just to create folders. Nothing to do with